### PR TITLE
feat: initial account experience docs

### PR DESCRIPTION
**Pull request by zepatrik**
All sections on the overview page could go into their own page with more content, but having the basics all on one page for now is already better than nothing.

### CODE REVIEW

File: docs/account-experience/index.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+sidebar_label: Account-Experience
+---
+
+The Ory Account Experience is the default user interface for all self-service screens like login, registration, or consent. It can
+be accessed under `https://your-slug.projects.oryapis.com/ui`. New Ory Network projects are automatically configured to use the
+Account Experience. You can find various customizations and settings in the Ory Console under `Account Experience`.
+
+## Theming
+
+The Account Experience can be themed using the Ory Console. You can find the theme settings under `Account Experience` >
```
Comment (aeneasr) [inline docs/account-experience/index.mdx:13]: Use a direct link :) This makes it so much easier to find if the descriptions ever change in the UI!

Here for example: https://console.ory.sh/projects/current/account-experience/theming

File: docs/account-experience/index.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+`Theming`. It is also possible to set a custom logo and favicon.
+
+## Welcome Screen
+
+The Account Experience comes with a welcome screen that shows information about the current user's session. As this screen is
+rather meant for debugging purposes, it can be disabled in the Ory Console under `Account Experience` > `UI URLs`.
```
Comment (aeneasr) [inline docs/account-experience/index.mdx:19]: link :)

**Review by aeneasr — COMMENTED** [review]
Nice!

**Review by aeneasr — DISMISSED** [review]

**Review by Benehiko — COMMENTED** [review]
Could you add a link to this page on here https://www.ory.sh/docs/kratos/bring-your-own-ui/custom-ui-overview#why-should-i-use-ory-account-experience?

**Review by vinckr — DISMISSED** [review]
lets go

**Pull request merged**